internal/corazarules/rule_match.go

```diff
@@ -125,7 +125,19 @@ func (mr MatchedRule) writeDetails(log *strings.Builder, matchData types.MatchDa
 		mr.ServerIPAddress_, mr.URI_, mr.TransactionID_))
 }
 
-func (mr MatchedRule) matchData(matchData types.MatchData, log *strings.Builder) {
+func (mr MatchedRule) writeExtraRuleDetails(log *strings.Builder, matchData types.MatchData, n int) {
+	msg := matchData.Message()
+	data := matchData.Data()
+	if len(msg) > 200 {
+		msg = msg[:200]
+	}
+	if len(data) > 200 {
+		data = data[:200]
```

Comment:
Can we please set this 200 to a constant like `maxSizeLogMessage` or whatever similar naming we think is good?